app/main.py

```diff
@@ -17,3 +17,15 @@
 collection_of_coins = {1, 2, 25}
 
 # write your code here
+def sort_variables(*args):
+    sorted_variables = {"mutable": [], "immutable": []}
+
+    for arg in args:
+        if isinstance(arg, (list, dict, set)):
+            sorted_variables["mutable"].append(arg)
+        else:
+            sorted_variables["immutable"].append(arg)
+
+    return sorted_variables
+
+print(sort_variables(lucky_number, pi, one_is_a_prime_number, name, my_favourite_films, profile_info, marks, collection_of_coins))
```

Comment:
Do not implement any additional logic for this task, you should only create a `sorted_variables` dict with two keys: `mutable` and `immutable`. You should have lists as values for these keys and populate them with corresponding variables. Besides that, you should fix flake8 errors upon the next review request.